dist/decanter-base.css

```diff
@@ -5,7 +5,7 @@
  * plugins.
  */
 
-/*! tailwindcss v2.2.15 | MIT License | https://tailwindcss.com
+/*! tailwindcss v2.1.2 | MIT License | https://tailwindcss.com
```

Comment:
@rebeccahongsf Is there a regression somewhere? Looks like TW version went from 2.2.15 to 2.1.2 😂  I was wondering why the huge # of line change.

The color change looks good to me otherwise! Thank you 😄

Reply:
Interesting....it looks like it went from 2.2.7 to 2.2.15 to @sherakama 's PR here
https://github.com/SU-SWS/decanter/pull/851/files

Hmm, wonder if we could just upgrade TW version 2.2.15 to match the one in the previous PR to avoid going back to an earlier 🤔 ?